deluge/httpdownloader.py

```diff
@@ -133,7 +133,9 @@ def request_callback(self, response):
                 content_disp = headers.getRawHeaders(b'content-disposition')[0].decode(
                     'utf-8'
                 )
-                content_disp_params = cgi.parse_header(content_disp)[1]
+                message = email.message.Message()
+                message['content-type'] = content_disp
```

Comment:
Why not use `content-disposition` and then `get_filename`

```
message['content-disposition'] = content_disp
new_file_name = message.get_filename()
if new_file_name:
    new_file_name = sanitise_filename(new_file_name)
```

Reply:
Thank you for comment. I did not notice this. Will reflect your review.